package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "cls-mongoose-shim",
-  "version": "0.0.8",
+  "name": "@adsk/cls-mongoose-shim",
+  "version": "0.0.9",
```

Comment:
bumping the subversion shouldn't be necessary because it's a new package name, however just to avoid any additional confusion on which package is which, i went ahead and bumped it... I honestly don't feel strongly one way or the other.